keras_cv/src/models/stable_diffusion_v3/clip_utils_test.py

```diff
@@ -91,8 +96,8 @@ def test_sd3_tokenizer(self):
         out_keras = sd3_tokenizer.tokenize_with_weights("A cat")
         expected_g_tokens = [(49406, 1.0), (320, 1), (2368, 1), (49407, 1.0)]
         expected_l_tokens = [(49406, 1.0), (320, 1), (2368, 1), (49407, 1.0)]
-        expected_t5xxl_tokens = [(71, 1), (1712, 1), (1, 1.0), (0, 1.0)]
+        expected_t5xxl_tokens = [(71, 1), (1712, 1), (1, 1.0), (0, 1.0)] # pylint: disable=unused-variable
```

Comment:
two spaces before addinf this ignore lint comment

Reply:
Added # noqa: F841